Source/WebKit/UIProcess/API/Cocoa/WKWebExtension.h

```diff
@@ -62,12 +62,12 @@ typedef NS_ERROR_ENUM(WKWebExtensionErrorDomain, WKWebExtensionError) {
     WKWebExtensionErrorInvalidBackgroundPersistence,
 } NS_SWIFT_NAME(WKWebExtension.Error) WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
 
-/*! @abstract This notification is sent whenever a @link WKWebExtension @/link has new errors or errors were cleared. */
+/*! @abstract This notification is sent whenever a ``WKWebExtension`` has new errors or errors were cleared. */
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA))
 WK_EXTERN NSNotificationName const WKWebExtensionErrorsWereUpdatedNotification NS_SWIFT_NAME(WKWebExtension.errorsWereUpdatedNotification);
 
 /*!
- @abstract A `WKWebExtension` object encapsulates a web extension’s resources that are defined by a `manifest.json` file.
+ @abstract A ``WKWebExtension`` object encapsulates a web extension’s resources that are defined by a `manifest.json`` file.
```

Comment:
```suggestion
 @abstract A ``WKWebExtension`` object encapsulates a web extension’s resources that are defined by a ``manifest.json`` file.
```

Reply:
Double backticks are really only for symbols. Code formatting should still be single backtick.

Reply:
> Double backticks are really only for symbols. Code formatting should still be single backtick.

OK, but that means `manifest.json` has an extra backtick at the end of it.

Reply:
```suggestion
 @abstract A ``WKWebExtension`` object encapsulates a web extension’s resources that are defined by a `manifest.json` file.
```